analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ClassShouldNotBeAbstract.CSharp11.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
```

Comment:
As far as I can tell, this rule does not check the type modifiers. I think it's enough to have one very simple noncompliant case and one compliant.